templates/web/base/report/new/duplicate_suggestions.html

```diff
@@ -6,9 +6,33 @@
     <button class="duplicate-report-suggestions__close js-hide-duplicate-suggestions">[% loc('Close') %]</button>
     <h2 class="form-section-heading">[% loc('Already been reported?') %]</h2>
     <div class="form-section-description">
-        <p>[% loc('We’ve already sent these similar nearby reports to the council. You can track them to receive updates.') %]</p>
+        <p>[% loc('We’ve already reported these similar problems nearby reports to the council. Is one of them yours?') %]</p>
     </div>
     <ul class="item-list"></ul>
-    <button class="btn btn--block js-hide-duplicate-suggestions">[% loc('No, continue with my new report') %]</button>
+    <button class="btn btn--block js-hide-duplicate-suggestions">[% loc('Continue – report a new problem') %]</button>
+</div>
+<div class="js-template-get-updates hidden">
+    <div id="alerts" class="get-updates">
+        <p id="rznvy_hint">
+            <strong>[% loc('Great!') %]</strong>
```

Comment:
Is it great? They've got an open problem, after all. Perhaps the last updated date is months ago and this makes it worse? :) I think we should drop this, anyway.

Reply:
Fair enough! Drop it.